molecule/default/verify.yml

```diff
@@ -12,5 +12,5 @@
 
   - assert:
       that:
-        - qdrlog_content is search('Configured Connector: stf-default-interconnect-5671-service-telemetry.example.com:443 .* role=edge')
+        - qdrlog_content is search('Configured Connector: stf-default-interconnect-5671-service-telemetry.example.com:443 proto=any, role=edge, sslProfile=sslProfile')
```

Comment:
No change here, but just FYI I've been using `infra.watch` for the domain in documentation. Probably no reason you couldn't use that if you didn't want `example.com` in there.

Reply:
RFC2606 says this is fine and I think I'd rather have this than a real domain, especially in live code.